include/MLX42/MLX42.h

```diff
@@ -567,6 +566,13 @@ bool mlx_loop_hook(mlx_t* mlx, void (*f)(void*), void* param);
  */
 mlx_texture_t* mlx_load_png(const char* path);
 
+/**
+ * @brief Uses the python script to convert from xpm into xpm42 and returns the new path
+ * 
+ * @param path the path of the file to convert
+ */
+char *convert_path(char* path);
+
```

Comment:
Please refer to the other functions as to how to provide the documentation and make sure to prefix your function with `mlx_`.

Also, make the path is a `const char*`, the return type needs it's star appended to the type. so `char*` not `char *`